packages/types/src/prop-type.ts

```diff
@@ -5,6 +5,7 @@ import type { CssProperties } from './system-types'
  * Shadowed export (in CLI): DO NOT REMOVE
  * -----------------------------------------------------------------------------*/
 
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
```

Comment:
🤔 if this is set to `"off"` in the `.eslintrc`, there shouldn't be a need to disable it inline in files.

ESLint's [`--report-unused-disable-directives`](https://eslint.org/docs/latest/use/command-line-interface#--report-unused-disable-directives) can help here.